src/components/Navigation.jsx

```diff
@@ -2,50 +2,47 @@ import React from 'react';
 import _ from 'lodash';
 import { Link } from 'react-router';
 
-export default class Navigation extends React.Component {
+export default function Navigation(props) {
+  // Render all categories in nav bar
+  let categories = ["off-campus", "on-campus", "commentary", "creative", "in-focus"];
+  let renderCategories =
+    _.map((categories || []), function(category) {
+      return(
+        <li key={category} className="navigation__categories__item">
+          <Link to={"/category/"+category} activeClassName="navigation__categories__item--active">
+            {category}
+          </Link>
+        </li>
+      )
+    });
 
-  render() {
-    // Render all categories in nav bar
-    let categories = ["off-campus", "on-campus", "commentary", "creative", "in-focus"];
-    let renderCategories =
-      _.map((categories || []), function(category) {
-        return(
-          <li key={category} className="navigation__categories__item">
-            <Link to={"/category/"+category} activeClassName="navigation__categories__item--active">
-              {category}
-            </Link>
-          </li>
-        )
-      });
-
-    return (
-      <div>
-        <div className="header">
-          <div className="header__search">
-            <input
-              className="header__search__main"
-              type="text"
-              placeholder="Search The Gazelle"
-            />
-            <div className="header__search__tail" />
-          </div>
-          <div className="header__title">
-            <Link to="/" className="header__title--content">{this.props.appName}</Link>
-          </div>
-          {/* TODO: change link to archives list */}
-          <Link to="/" className="header__issueId">{"Issue 55"}</Link>
+  return (
+    <div>
+      <div className="header">
+        <div className="header__search">
```

Comment:
On the topic of search, however, right now the search box only appears when the mouse hovers over the magnifying glass. The plan is to expand the search box when the user hovers over "SEARCH" as well. It's only my list of things to do